backends/p4tools/cmake/common.cmake

```diff
@@ -0,0 +1,49 @@
+# Common configuration across all P4Testgen projects.
```

Comment:
so is this P4tools or p4testgen? I prefer the latter.

Reply:
It is P4Tools. P4Testgen is one tool contained within P4Tools. Currently, the only one but there will be more.